api/users.go

```diff
@@ -0,0 +1,180 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/ONSdigital/dp-identity-api/api/content"
+	"github.com/ONSdigital/dp-identity-api/apierrors"
+	"github.com/ONSdigital/dp-identity-api/models"
+	"github.com/ONSdigital/dp-identity-api/validation"
+	"github.com/ONSdigital/log.go/log"
+	"github.com/aws/aws-sdk-go/service/cognitoidentityprovider"
+	"github.com/google/uuid"
+	"github.com/sethvargo/go-password/password"
+)
+
+var errDuplicateEmail = errors.New("duplicate email")
```

Comment:
For consistency this could go in apierrors with the other errors.

Reply:
Ok; will move it. Left it in there as its unlikely to be needed by any other package,